src/js/background/assignManager.js

```diff
@@ -764,7 +764,7 @@ window.assignManager = {
   async initBookmarksMenu() {
     browser.contextMenus.create({
       id: this.OPEN_IN_CONTAINER,
-      title: "Open Bookmark in Container Tab",
+      title: browser.i18n.getMessage("openBookmarkInContainerTab"),
```

Comment:
Where do I see this string? I have the add-on installed, but don't see it in the context menu for bookmarks.

Reply:
1. In MAC preferences, you need to tick "Enable bookmark menus"

![image](https://github.com/mozilla/multi-account-containers/assets/7339076/3c140545-f857-481f-bf88-be5bca8ed058)

2. Open Firefox sidebar and select the bookmarks view (Ctrl + B).
3. Right-Click on a bookmark